qtfred/src/ui/dialogs/ShipEditorDialog.cpp

```diff
@@ -156,6 +156,16 @@ ShipEditorDialog::ShipEditorDialog(FredView* parent, EditorViewport* viewport)
 
 ShipEditorDialog::~ShipEditorDialog() = default;
 
+int ShipEditorDialog::getShipClass()
+{
+	return _model->getShipClass();
+}
+
+bool ShipEditorDialog::getMulti()
```

Comment:
Multi has a lot of different meanings: multiplayer, multicontroller.  If you could make this a little more specific for readability that would be good.